usr/share/rear/backup/DUPLICITY/default/50_make_duplicity_backup.sh

```diff
@@ -1,7 +1,7 @@
 # This file is part of Relax-and-Recover, licensed under the GNU General
 # Public License. Refer to the included LICENSE for full text of license.
 
-if [ "$BACKUP_PROG" = "duply" ] && has_binary duply; then
+if [ "$BACKUP_PROG" = "duply" ] && has_binary duplyi ; then
```

Comment:
@gdha 
you should not look only at the initial commit
(I detected this erreor and fixed it with my second commit)
better use the "Files changed" tab at the top because
that shows the resulting changes of all my commits.